### PR TITLE
Nikhil info

### PR DESCRIPTION
**Pull request by nikhil1007**
Pull request created as part of Class Assignment-1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,20 +19,19 @@
...
-			      <li>Your major
-		              <li>Shawshank's Redemption
-		  	     </ul>
+		        <li>Bangalore, India
+			      <li>Masters in Information Systems and Managament
+		        <li>Coming to America- Eddie Murphy
+		  	  </ul>
 		    </section>
 		    <section>
-				copy and paste yours
```
Comment (shots47s) [inline index.html:35]: You did not follow the instructions, I asked you to copy the slide to here, please conform to that instruction.

File: index.html [hunk 2]
```diff
@@ -19,20 +19,19 @@
 
 	<body>
 		<div class="reveal">
-                  <div class="slides">
+    <div class="slides">
```
Comment (shots47s) [inline index.html]: This is an extraneous change, please restore to the original format.

**Review by shots47s — CHANGES_REQUESTED** [review]
See comments in the file.

**Comment by nikhil1007** [comment]
I had a lot of errors working with the branch "nikhil_info", I deleted it and have submitted a new pull request.

**Pull request closed without merging**